### DISCUSSION TITLE
Install, anaconda mac

### DISCUSSION THREAD

**Discussion by DHLynch** [isl-org/Open3D#6262] [General]
Hi all, I am trying to use the advised command:
`conda install -c open3d-admin open3d`

However I get thrown this:

`PackagesNotFoundError: The following packages are not available from current channels:

  - open3d

Current channels:

  - https://conda.anaconda.org/open3d-admin/osx-arm64
  - https://conda.anaconda.org/open3d-admin/noarch
  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://conda.anaconda.org/conda-forge/osx-arm64
  - https://conda.anaconda.org/conda-forge/noarch`

Any help would be appreciated, thanks in advance

**Comment by cwreynolds** [comment]
I am no authority on any of this, but in case it helps get you unstuck, this describes how I dealt with a similar issue back in March: https://github.com/isl-org/Open3D/discussions/6032

**Reply by DHLynch** [reply]
Hi, @cwreynolds really appreciate your response. I figured it was something like that, but I quickly went into a deep rabbit hole trying to install the missing package, where I then didn't have brew, brew install failed with errors, and sat there thinking I've got to be missing something obvious!

**Reply by DHLynch** [reply]
As an update, I managed to overcome all hurdles and solved the issues the same way @cwreynolds did. Thanks!
